sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,467 @@
+(235434, 1, 330.012, -4760.3, 25.6922, 0, 0, 0, 0, 100, 0),
+(235434, 2, 304.91, -4765.39, 15.8638, 0, 0, 0, 0, 100, 0),
+(235434, 3, 301.86, -4743.94, 20.7011, 0, 0, 0, 0, 100, 0),
+(235434, 4, 309.97, -4716.32, 24.9401, 0, 0, 0, 0, 100, 0),
+(235434, 5, 326.66, -4709.62, 27.1919, 0, 0, 0, 0, 100, 0),
+(235434, 6, 345.85, -4713.88, 29.6655, 0, 0, 0, 0, 100, 0),
+(235434, 7, 368.08, -4705.2, 32.3836, 0, 0, 0, 0, 100, 0),
+(235434, 8, 374.094, -4720.59, 30.957, 0, 0, 0, 0, 100, 0),
+(235434, 9, 325.143, -4733.62, 35.8289, 0, 0, 0, 0, 100, 0);
+
+-- Brill
+DELETE FROM `waypoint_data` WHERE `id`=235435;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(235435, 1, 2219.06, 281.009, 52.45, 0, 0, 0, 0, 100, 0),
+(235435, 2, 2241.25, 297.52, 43.7327, 0, 0, 0, 0, 100, 0),
+(235435, 3, 2274.95, 305.308, 38.4187, 0, 0, 0, 0, 100, 0),
+(235435, 4, 2278.97, 299.66, 52.3487, 0, 0, 0, 0, 100, 0),
+(235435, 5, 2288.88, 280.746, 67.0288, 0, 0, 0, 0, 100, 0),
+(235435, 6, 2269.29, 268.162, 56.6651, 0, 0, 0, 0, 100, 0),
+(235435, 7, 2267.19, 239.023, 56.9931, 0, 0, 0, 0, 100, 0),
+(235435, 8, 2238.28, 236.233, 55.7377, 0, 0, 0, 0, 100, 0),
+(235435, 9, 2228.26, 253.414, 49.4476, 0, 0, 0, 0, 100, 0),
+(235435, 10, 2241.27, 275.894, 41.3221, 0, 0, 0, 0, 100, 0),
+(235435, 11, 2263.52, 296.479, 65.8, 0, 0, 0, 0, 100, 0),
+(235435, 12, 2258.47, 292.451, 62.7599, 0, 0, 0, 0, 100, 0);
+
+-- Falconwing Square
+DELETE FROM `waypoint_data` WHERE `id`=235436;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(235436, 1, 9541.06, -6838.29, 21.2302, 0, 0, 0, 0, 100, 0),
+(235436, 2, 9528.65, -6852.61, 31.3788, 0, 0, 0, 0, 100, 0),
+(235436, 3, 9514.61, -6853.57, 31.1926, 0, 0, 0, 0, 100, 0),
+(235436, 4, 9498.61, -6836.72, 30.143, 0, 0, 0, 0, 100, 0),
+(235436, 5, 9485.69, -6820.25, 29.7564, 0, 0, 0, 0, 100, 0),
+(235436, 6, 9494.28, -6802.62, 23.1539, 0, 0, 0, 0, 100, 0),
+(235436, 7, 9515.9, -6789.68, 30.1268, 0, 0, 0, 0, 100, 0),
+(235436, 8, 9535.82, -6795.01, 26.1349, 0, 0, 0, 0, 100, 0),
+(235436, 9, 9531.41, -6812.67, 22.2707, 0, 0, 0, 0, 100, 0),
+(235436, 10, 9515.89, -6826.04, 28.0319, 0, 0, 0, 0, 100, 0),
+(235436, 11, 9508.7, -6812.54, 41.3166, 0, 0, 0, 0, 100, 0);
+
+DELETE FROM `creature_text` WHERE `entry`=23543;
+INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(23543, 0, 0, "Prepare yourselves, the bells have tolled! Shelter your weak, your young and your old! Each of you shall pay the final sum. Cry for mercy, the reckoning has come!", 14, 0, 100, 0, 0, 11966, 22022, 0, "lol"),
```

Comment:
That's a leftover I missed during cleanup phase :P not intended. Thanks for the notice.